File: 12_Swap two numbers/swap.py
# Swapping content of two variables without using any other variable.

a,b = int(input("Enter first number: ")), int(input("Enter second number: "))

print("Initial: A=", a,"B=", b)

a = a+b
b = a-b
a = a-b

'''
for Example:
    a=100
    b=50

    a = a+b [ a=150 ]
    b = a-b [ b=100 ]
    a = a-b [ a=50 ]
'''

print("Final: A=", a,"B=", b)

File: 12_Swap two numbers/swap.c
// Swapping content of two variables without using any other variable.

#include<stdio.h>

int main(int argc, char const *argv[])
{
    int a, b;

    printf("Enter A: ");
    scanf("%d", &a);
    printf("Enter B: ");
    scanf("%d", &b);

    printf("Initial A: %d B: %d\n", a, b);

    a=a+b;
    b=a-b;
    a=a-b;

    printf("Final A: %d B: %d", a, b);
    return 0;
}

